### PR TITLE
Include native binaries

### LINKED ISSUE #29: MSIX Support
Figure out how to get this working with MSIX and/or what the requirements are.

### PR DESCRIPTION
**Pull request by onovotny**
This should let the library and .net core global tool be self-contained. It carries the x86 and x64 files alongside and loads the right one per platform. 

The one breaking change is that `AuthenticodeKeyVaultSigner.Initialize();` must now be called early on by the host.

Based on the loaded modules, I believe everything is working correctly, but it would be useful to test an MSIX on a pre-RS5 Win10 machine to be sure.

/cc @riverar

Replaces #19 
Fixes #29

### CODE REVIEW

File: src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs [hunk 1]
```diff
@@ -272,5 +278,47 @@ public void Dispose()
...
+        /// <param name="assemblyPath">Override the default location for the assembly. Necessary when running on Azure App Services</param>
+        public static void Initialize(string assemblyPath = null)
+        {
+            if (ManifestLocation != null)
+                return; // already initialized
+
+
+            var is64bit = IntPtr.Size == 8;
+
+            if(!string.IsNullOrWhiteSpace(assemblyPath))
```
Comment (vcsjones) [inline src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs]: Did you mean to assign this if it _is_ null or whitespace?

File: src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs [hunk 2]
```diff
@@ -20,7 +21,7 @@ public class AuthenticodeKeyVaultSigner : IDisposable
         private readonly MemoryCertificateStore _certificateStore;
         private readonly X509Chain _chain;
         private readonly SignCallback _signCallback;
-
+        private static string ManifestLocation;
```
Comment (vcsjones) [inline src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs]: Should be named `_manifestLocation`.

File: src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs [hunk 3]
```diff
@@ -272,5 +278,47 @@ public void Dispose()
...
+        /// The initialize should happen as early as possible in the host process.
+        /// </summary>
+        /// <param name="assemblyPath">Override the default location for the assembly. Necessary when running on Azure App Services</param>
+        public static void Initialize(string assemblyPath = null)
+        {
+            if (ManifestLocation != null)
+                return; // already initialized
+
+
+            var is64bit = IntPtr.Size == 8;
```
Comment (vcsjones) [inline src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs]: Can we use [`Environment.Is64BitProcess`](https://docs.microsoft.com/en-us/dotnet/api/system.environment.is64bitprocess?view=netcore-2.1)? There's nothing "wrong" with this code but it seems to express the intent better to a reader.

File: src/AzureSign.Core/README.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+﻿# Redist of files in x64/x86
+
+The files in the x86/x64 directory are covered by the redist in the Windows SDK:
+http://go.microsoft.com/fwlink/?LinkId=524842
```
Comment (vcsjones) [inline src/AzureSign.Core/README.md]: HTTPS link please :-).

File: src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs (hunk 1 above)
Comment (onovotny) [reply src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs]: Yeah, fixed.

File: src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs (hunk 2 above)
Comment (vcsjones) [reply src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs]: Scratch that, it's static.

File: src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs (hunk 3 above)
Comment (onovotny) [reply src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs]: I've never been sure if one of them lies?

Comment (vcsjones) [reply src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs]: It does the exact same thing as your code.

https://github.com/dotnet/corefx/blob/28e68e5426332a0f45c5fa5b2c221f74d3bed032/src/System.Runtime.Extensions/src/System/Environment.cs#L141

**Comment by vcsjones** [comment]
LGTM. I will merge and play with it sometime Monday on a few OSes to check that everything works. If all seems good I will put a prerelease up on nuget Monday evening. Thanks for getting this over the line!

**Comment by onovotny** [comment]
If you have it available, a good test is Server 2016....as that's what App Services is running and is an older "Win10" variant.

**Comment by riverar** [comment]
I can test this on pre-RS4, standby.

**Pull request merged**